node_server/src/main/models/task.ts

```diff
@@ -25,6 +25,13 @@ const taskSchema = new Schema({
     type: require('mongoose-int32'),
     default: 0,
   },
+  prereqTasks: [
+    {
+      type: ObjectId,
+      ref: 'Task',
+      default: new Array<typeof ObjectId>(),
+    },
+  ],
```

Comment:
Does this need to be defined for projects well?

Reply:
Also this will need to be included in the `swagger.ts` file at some point too.

Reply:
Did u mean as well? U said that u wanted to give projects prerequisites too. Just haven't gotten around to using it.

Ok. ill try to figure that out.

Reply:
Im out of time. I cant get projects in too. it shouldn't be too difficult to implement for projects? Just not sure how that will work exactly.